duckduckgo/BrowserServicesKit
# Add privacy config version to broken site reports

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/414709148257752/1208158714445706/f
iOS PR: https://github.com/duckduckgo/iOS/pull/3351
macOS PR: https://github.com/duckduckgo/macos-browser/pull/3256
What kind of version bump will this require?: Major/Minor/Patch

**Optional**:

Tech Design URL:
CC:

**Description**:



**Steps to test this PR**:
1. Submit a broken site report
1. Set a breakpoint where the report is created or look at the logs for the pixel
2. Ensure the `remote_config_version` parameter is present and filled with the config version



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
